frontend/server/src/Controllers/ACL.php

```diff
@@ -62,4 +62,118 @@ public static function removeGroup(
             'role_id' => $roleId,
         ]));
     }
+
+    /**
+     * Returns all ACLs owned by the current user and the roles assigned within those ACLs.
+     *
+     * @param \OmegaUp\Request $r The request object containing user session information.
+     * @throws \OmegaUp\Exceptions\NotFoundException If the user is not found.
+     * @return array{acls: list<array{acl_id: int, type: string, alias: ?string}>, roles: list<array{acl_id: int, user_id: int, username: string, role_id: int, role_name: string, role_description: string}>}
+     */
+    public static function apiUserOwnedAclReport(\OmegaUp\Request $r): array {
+        // Ensure user is authenticated
+        $r->ensureMainUserIdentity();
+
+        /** @var null|\OmegaUp\DAO\VO\Identities $user */
+        $user = $r->identity;
```

Comment:
For consistency:
```suggestion
        $identity = $r->identity;
```
If you're interested in the user, you can access it directly via `$r->user`